CHANGELOG.md

```diff
@@ -20,8 +20,16 @@ Line wrap the file at 100 chars.                                              Th
 * **Fixed**: for any bug fixes.
 * **Security**: in case of vulnerabilities.
 
-
 ## [Unreleased]
+
+### Added
+
+### Changed
+
+### Removed
```

Comment:
IMHO you don't need to add headers for things that have not yet happened. It could be that the next release only has `Added` and none of the other headers. So they can be added as commits come in. That's how I have seen it used most of the time.

Sidenote: Iff you wanted to be exhaustive from the get-go you are missing `Deprecated`, `Fixed` and `Security`,